pkg/skuba/actions/cluster/init/init.go

```diff
@@ -75,13 +79,20 @@ func (initConfiguration InitConfiguration) ControlPlaneHostAndPort() string {
 func NewInitConfiguration(clusterName, cloudProvider, controlPlane, kubernetesDesiredVersion string, strictCapDefaults bool) (InitConfiguration, error) {
 	kubernetesVersion := kubernetes.LatestVersion()
 	var err error
+	needsHyperKube := false
```

Comment:
nit: Boolean default value is false in Golang.

Reply:
so you prefer `var needsHyperKube bool` ? Fun story, this is what I had at the previous iteration.
I would rather avoid changing my code on style again on this, except if we have a consensus.

Note: I prefer being explicit. Having false written somewhere is important for my reviewing :)